sceptre/cli/__init__.py

```diff
@@ -45,10 +45,21 @@
     help="A YAML file of variables to template into config files.")
 @click.option(
     "--ignore-dependencies", is_flag=True, help="Ignore dependencies when executing command.")
+@click.option(
+    "--j2_extension", "j2_extensions",  multiple=True,
```

Comment:
would it make sense to pass in a dict that can override any of the existing jinja2 environment settings including the new jinja2.extensions setting?

Reply:
Finally sat down to look into this again.I like the idea of passing all the jinj2 extensions.
@zaro0508  What are your opinions on passing in a dict via the CLI (as currently implemented) vs having it reside in the stack/stackgroup config?